tests/README.md

```diff
@@ -0,0 +1,72 @@
+## Backend development
+
+The backend test suite can be run locally. 
+
+### Start the services
+
+We need all but the main `backend` service running.  Since multiple `backend` instances are allowed, we will simply start everything up
+
+    # ENSURE THE IMAGES ARE CLEAN
+    docker-compose down
+    # SETUP ALL IMAGES
+    docker-compose up --build
+
+### Install into virtual environment
+
+Treeherder, Django, and the required libraries will ruin you local Python install. It is best to setup a virtual environment to contain the quirks it introduces.
+
+Be sure you are in the `treeherder` main directory
+
+    pip install virtualenv
+    virtualenv venv
+    source venv/bin/activate
+    venv/bin/pip install -r tests/requirements.txt
```

Comment:
Approving this new requirements file will mean accepting that `tests/requirements.txt` will fall out of date as new dependencies are added.

Reply:
This will also mean that local testing might not yield the same results as in Travis but I'm willing to take the risk.

Reply:
Yes, I could not get the existing requirements files to work so this one was made.  There is something about `pip install -r` that does not always install all the listed libraries.  Anyway, the hope is this code is not part of a critical path, and is not even necessary to pass tests; it is only optional to run locally and improve the debug cycle.